rbd/rbd_test.go

```diff
@@ -1767,3 +1767,68 @@ func TestRadosIoctxInvalid(t *testing.T) {
 		_ = cephIoctx(ioctx)
 	})
 }
+
+func TestDataPoolGetId(t *testing.T) {
```

Comment:
The word order in this test function name is different from the word order in the new function name:  "test data pool get id" vs. "get data pool id". If this function exists to test the new function it should share the name. IOW - `TestGetDataPoolID`.

Reply:
Do not add this function to an existing file. Create a new file `rbd/data_pool_id_test.go` that has a matching `go:build` line.